src/api_client/api_client.js

```diff
@@ -37,8 +38,25 @@ const defaultOptions = {
 };
 
 const commonHeaders = {
+	Accept: 'application/json',
 	'Content-Type': 'application/json',
-	os: 'lisk-js-api',
+};
+
+const getUserAgent = (
+	{ name = '????', version = '????', engine = '????' } = {},
+) => {
+	const liskJSInformation = 'LiskJS/1.0 (+https://github.com/LiskHQ/lisk-js)';
+	const locale =
+		process.env.LC_ALL ||
+		process.env.LC_MESSAGES ||
+		process.env.LANG ||
+		process.env.LANGUAGE;
+	const systemInformation = `${os.platform()} ${os.release()}; ${os.arch()}${
+		locale ? `; ${locale}` : ''
+	}`;
+	return `${name}/${version} (${engine}) ${liskJSInformation} ${
+		systemInformation
+	}`;
```

Comment:
There should be a `,` between lisk and system information.

Reply:
Do you have any examples of this as a standard? It doesn't appear anywhere here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent